### ISSUE TITLE
Modificar  PersistenceManager y ExcelReportManager

### ISSUE THREAD

**Issue by Isabel-Roman** [MIT-FS/Audit4Improve-API#35]
Modificar para que sea acorde al modelo:
![controlPackage-V 0 2](https://user-images.githubusercontent.com/3404275/165074344-ffa72196-bdbb-4191-9e61-cc2598d38a40.GIF)
El método que persiste el indicador, antes de guardar en el fichero, comprueba el estado y la fuente cumpla con lo que indique ReportFormaterI
(Checker.getIndicatorState)

Repartir métodos y tests en la pareja

**Comment by ghost** [comment]
Para obtener la fuente del Indicator necesito saber el estado del Indicator, pero no exisite método para ello, cómo lo hago? Concretamente es en la llamada a formater.getIndicatorFont(indicator.getState()), ese método getState() no existe, y no puedo acceder a la propiedad State por que es privada...

**Comment by ghost** [comment]
Adicionalmente, para cambiar el estilo del workbook de excel, uso style.setFont() pero el argumento que usa es un objeto de tipo (org.apache.poi.ss.usermodel.Font) y no (java.awt.Font), como está definido en ReportFormartI. Se podría usar este primer tipo?

**Comment by Isabel-Roman** [comment]
Respecto a getState, sí existe, mira el issue 33...
No está implementado todavía, pero sí existe. Tú tienes que usarlo, como si estuviera ya implementado

**Comment by Isabel-Roman** [comment]
Respecto a cambiar el tipo de Font, no hay problema, pero tienes que informarlo adecuadamente cuando hagas la entrega. En el PR y con comentarios en el código (javadoc)

**Comment by ghost** [comment]
Ah, esperaba que al menos estuviera en la interfaz pero no lo ví, muchas gracias. Lo del Font te preguntaré proximamente

**Comment by Isabel-Roman** [comment]
Sí, me dices qué quieres cambiar y vemos dónde hacerlo, puede que sea más adecuado hacer una transformación de clases
